### ISSUE TITLE
Access Denied while uploading?

### ISSUE THREAD

**Issue by PRinceBhai** [yapb/graph#60]
I made a graph for a map in a week and now that I've done correcting it, I tried to upload it but I get the following message :

```
Something went wrong with uploading. Come back later. (AccessDenied)


There is an internal error, or something is totally wrong with
your files, and they are not passed sanity checks. Sorry...
```

While the Graph passes the Save Check.

**Comment by PRinceBhai** [comment]
So I found out that the test .dll that I used from the previous issues have different version or workings of the graphs. Which causes them to not get uploaded or read by the previous versions. :|

**Comment by jeefo** [comment]
it's due to graph version bump, will try to fix it next week

**Comment by jeefo** [comment]
and, can you verify if old versions of yapb, can read graphs with 'modified_by' extension,pls

**Comment by PRinceBhai** [comment]
Tried from this build https://github.com/yapb/yapb/issues/355 (VIP Bot picking up weapons)

The cvar `yb graph fileinfo` isn't even registered it seems, it just opens the graph menu.

**Comment by Vladislav4KZ** [comment]
> and, can you verify if old versions of yapb, can read graphs with 'modified_by' extension,pls

@jeefo old versions of YaPB don't read the new 3-rd version of graph. Because of this, it is not possible to check the compatibility of graphs with the ``modified_by`` extension on older versions of the bot.

This is what it wrote in the console
```
Damaged Graph (filename: 'de_dust.graph'). Version number differs (got: '3', need: '2').
```

Tested on build from GitHub Actions: https://github.com/yapb/yapb/actions/runs/3081362221
```
YaPB v4.2.700 (ID 700:cfa0277)
от YaPB Project (yapb@jeefo.net)
   https://yapb.jeefo.net/
скомпилировано: Sep 19 2022 09:36:10 на fv-az319-951 с помощью msvc 19.33.31629
```

**Comment by Vladislav4KZ** [comment]
> and, can you verify if old versions of yapb, can read graphs with 'modified_by' extension,pls

@jeefo I saved the waypoint in the latest build of YaPB from GitHub Actions (with reverted graph version), then I checked the compatibility of the graph with the old version of YaPB. And it works! The older version of YaPB successfully read the graph file with the ``modified_by`` extension.

**Comment by Vladislav4KZ** [comment]
Is this issue still relevant?

**Comment by PRinceBhai** [comment]
> Is this issue still relevant?

Yes I needed a updated win32 binary to upload maps. I still can't upload maps.

**Comment by Vladislav4KZ** [comment]
> Yes I needed a updated win32 binary to upload maps. I still can't upload maps.

@PRinceBhai I am using the latest build from GitHub Actions ([v4.2.711](https://github.com/yapb/yapb/actions/runs/3302643876)), and graph uploading works. You can see that I just uploaded the de_halloween_night.graph to database

What version do you have?

jeefo said that the graphs could not be uploaded to the database due to the new version of the graph (Graph v3.0). But the latest build from GitHub Actions has a reverted graph version (Graph v2.0, but with extensions support: author name of original graph, and author name of modified graph version)

**Comment by PRinceBhai** [comment]
> @PRinceBhai I am using the latest build from GitHub Actions ([v4.2.711]

I just tried this build, but I still can't upload, I get the same error :

Something went wrong with uploading. Come back later. (AccessDenied)

There is an internal error, or something is totally wrong with
your files, and they are not passed sanity checks. Sorry...

Also when I use "yp graph fileinfo" yapb is not showing the "author" and "modified by" tags. It just gives a blank result on the waypoint I've made.

**Comment by Vladislav4KZ** [comment]
@PRinceBhai can you give me your graph file? What version/build of yapb was it made on?

In the latest version of YaPB, the problem with nicknames in ``yb g fileinfo`` has been fixed.

You can try to re-save the graph on the new version of YaPB and check for the presence of a nickname in ``author:`` string from ``yb g fileinfo`` output

**Comment by PRinceBhai** [comment]
I was using this build :
https://github.com/yapb/yapb/actions/runs/3093246833

I've attached a graph I made with this build.

[dm_gallery_cso.zip](https://github.com/yapb/graph/files/9943432/dm_gallery_cso.zip)

**Comment by Vladislav4KZ** [comment]
> I've attached a graph I made with this build.
> 
> [dm_gallery_cso.zip](https://github.com/yapb/graph/files/9943432/dm_gallery_cso.zip)
> 

Ok, I'll take a look later.

To create new graphs, use the latest build of YaPB, to avoid old issues.

**Comment by Vladislav4KZ** [comment]
@PRinceBhai take a look: https://github.com/yapb/graph/commit/7f6329c867f1d311c7995faab153938fd92fa316

I looked at the output of the ``yb g fileinfo`` command. It says that the version of this graph is 3.0
You couldn't upload the graph due to Graph v3.0 format (previous version of YaPB)

I resaved your graph file on latest version of YaPB, so that it is in Graph v2.0 format. Original author name is saved.
